### PR TITLE
chore: update to v10

### PR DESCRIPTION
**Pull request by alisonjoseph**
Updated `carbon-components`, `carbon-components-react` and `carbon-elements` (except icons) to latest rc release

- renamed DropDownV2 to DropDown

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,14 +39,14 @@
     "@carbon/icons-react": "0.0.1-alpha.31",
     "@mdx-js/mdx": "^0.16.6",
     "@mdx-js/tag": "^0.16.6",
-    "@carbon/colors": "beta",
-    "@carbon/grid": "beta",
-    "@carbon/import-once": "beta",
-    "@carbon/layout": "beta",
-    "@carbon/type": "beta",
+    "@carbon/colors": "v10.0.0-rc.0 (",
```
Comment (asudoh) [inline package.json]: Did you mean to remove `(`?

**Comment by alisonjoseph** [comment]
@joshblack need your help on this also. getting errors updating to latest React

Screen Shot 2019-03-28 at 10 01 12 AM

**Comment by jnm2377** [comment]
Looks like the pagination component still has `v2` in it and it's causing the build to break.

Should this be updated too?
```
---
title: Pagination
tabs: ['Code', 'Usage', 'Style']
---

<ComponentCode
  name="Pagination"
  component="pagination"
  variation="pagination--v2"
  hasReactVersion
/>
<ComponentCode
  name="Pagination disabled buttons"
  component="pagination"
  variation="pagination--v2-disabled-pagination-buttons"
  hasReactVersion
/>
<ComponentDocs component="pagination" />
```

**Comment by lovemecomputer** [comment]
seeing a couple of pages fail to load:
- `/guidelines/themes/`
- `/guidelines/layout/`

(i'm assuming there are a couple others)

sometimes it's just a blank page, other times an error screen:
Screen Shot 2019-03-29 at 2 15 22 PM

**Pull request merged**